setup.py

```diff
@@ -1,17 +1,21 @@
 from setuptools import setup, find_packages
+import os
 
-with open("requirements.txt") as f:
+install_folder = os.path.dirname(os.path.realpath(__file__))
+requirementPath = install_folder + '/requirements.txt'
+with open(requirementPath) as f:
     install_requires = f.read().splitlines()
 
 setup(
-    name="perceiver-pytorch",
+    name="perceiverio",
```

Comment:
You happy with the name?

Reply:
I'm not sure its the best name, but `perceiver` is taken, same with a lot of other shortish names. But totally open to any other suggestions!

Reply:
yea, maybe perceiverio is ok. It has some none perceiver io stuff in it, so i dont know how to reflect that. Maybe perceiverio is good, and I'm over thinking it

Reply:
Yeah, `perceiverio` is OK. 

`perceiver-io` is perhaps more readable, and Pip is happy with hyphens.

Something like `perceivermodel` or `perceiver-model` might be worth considering:  It's more obviously two words (instead of 
`perceiverio` which is all-too-tempting to read as a single word: 'perceivereo') and it is perhaps more general than `perceiverio`, so it consistent with the fact that the code includes both `perceiver` and `perceiver IO` code.  But maybe I'm over-thinking this :)

Reply:
Okay, yeah, I'll change it to `perceiver-model`, sounds like a good way to make it sound more general!